go/pkg/bertymessenger/db.go

```diff
@@ -470,10 +471,15 @@ func (d *dbWrapper) addContactRequestOutgoingSent(contactPK string) (*messengert
 		}); res.Error != nil {
 		return nil, res.Error
 	} else if res.RowsAffected == 0 {
-		return nil, errcode.ErrDBAddContactRequestOutgoingSent.Wrap(fmt.Errorf("nothing found"))
+		// Check if that not already in the DB.
+		res := d.db.Where(&messengertypes.Contact{PublicKey: contactPK}).First(&contact)
```

Comment:
hum, I think we need someone else to check if this not a problem, because we are explicitly taking a contact with any State

can someone else give a look please?